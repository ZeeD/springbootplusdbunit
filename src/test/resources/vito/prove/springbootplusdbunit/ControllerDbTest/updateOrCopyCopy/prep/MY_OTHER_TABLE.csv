id,name,when
10,null,1234-05-06 07:08:09
