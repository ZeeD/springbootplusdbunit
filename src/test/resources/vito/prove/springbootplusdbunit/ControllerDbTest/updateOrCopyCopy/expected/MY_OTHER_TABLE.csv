id,name,something_date
1,"name",1982-11-05 05:58:13 +0000
10,null,1234-05-06 07:08:09
